src/Aspire.Hosting.Redis/RedisBuilderExtensions.cs

```diff
@@ -235,218 +229,49 @@ public static IResourceBuilder<RedisResource> WithRedisInsight(this IResourceBui
-                        // a conflicting name so we delete them all. This just keeps
-                        // track of the specific ID that we need to delete.
-                        databasesToDelete.AddRange(lookup[redisResource.Name]);
-                    }
-
-                    if (redisResource.PrimaryEndpoint.IsAllocated)
-                    {
-                        var endpoint = redisResource.PrimaryEndpoint;
-                        writer.WriteStartObject();
-
-                        writer.WriteString("host", redisResource.Name);
-                        writer.WriteNumber("port", endpoint.TargetPort!.Value);
-                        writer.WriteString("name", redisResource.Name);
-                        writer.WriteNumber("db", 0);
-                        writer.WriteNull("username");
-                        if (redisResource.PasswordParameter is { } passwordParam)
-                        {
-                            writer.WriteString("password", passwordParam.Value);
-                        }
-                        else
-                        {
-                            writer.WriteNull("password");
-                        }
-                        writer.WriteString("connectionType", "STANDALONE");
-                        writer.WriteEndObject();
-                    }
-                }
-                writer.WriteEndArray();
-                await writer.FlushAsync(cancellationToken).ConfigureAwait(false);
-                stream.Seek(0, SeekOrigin.Begin);
-
-                var content = new MultipartFormDataContent();
+                    var counter = 1;
 
-                var fileContent = new StreamContent(stream);
-
-                content.Add(fileContent, "file", "RedisInsight_connections.json");
-
-                var apiUrl = $"{databasesPath}/import";
-
-                try
-                {
-                    if (databasesToDelete.Any())
+                    foreach (var redisInstance in redisInstances)
                     {
-                        await pipeline.ExecuteAsync(async (ctx) =>
+                        if (redisInstance.PrimaryEndpoint.IsAllocated)
                         {
-                            // Create a DELETE request to send to the existing instance of
-                            // RedisInsight with the IDs of the database to delete.
-                            var deleteContent = JsonContent.Create(new
-                            {
-                                ids = databasesToDelete
-                            });
-
-                            var deleteRequest = new HttpRequestMessage(HttpMethod.Delete, databasesPath)
+                            // RedisInsight assumes Redis is being accessed over a default Aspire container network and hardcodes the resource address
+                            // This will need to be refactored once updated service discovery APIs are available
```

Comment:
@danegsta - is this comment still relevant?

Reply:
It's technically relevant, but I'd say only the first line of the comment is really adding any value.